client/src/Components/App/App.js

```diff
@@ -1,5 +1,5 @@
 import React, {useState, useEffect} from 'react';
-import { Route, NavLink, Switch } from 'react-router-dom';
+import { Route, Switch } from 'react-router-dom';
```

Comment:
Haha yes!! I also deleted this in my last PR - NO UNUSED IMPORTS GANG

Reply:
Yes! Thanks for the cleanup!